Node.gitignore

```diff
@@ -26,6 +26,14 @@ coverage
 # nyc test coverage
 .nyc_output
 
+#cypress
+cypress/integration/1-getting-started/
+cypress/integration/2-advanced-examples/
```

Comment:
These files seem specific to a tutorial or guide. They are not listed in the documentation you provided.

Reply:
- Could not find any documentation on Cypress seeding the project with these files but they are scaffolded once you run Cypress initially. 
- You do have options to remove them, so adding the dir to the ignore list might not always be useful. 

What do you think? 
<img width="1093" alt="Screenshot 2021-12-05 at 12 50 16" src="https://user-images.githubusercontent.com/11727537/144745509-420d0bdf-6228-4fca-8e5d-1f4f53ed7e60.png">

Reply:
- Could not find any documentation on Cypress seeding the project with these files but they are scaffolded once you run Cypress initially. 
- You do have options to remove them, so adding the dir to the ignore list might not always be useful. 
- However I don't assume anyone cares to stage those. 

What do you think? 
<img width="1093" alt="Screenshot 2021-12-05 at 12 50 16" src="https://user-images.githubusercontent.com/11727537/144745606-0d756b28-1fee-4063-b477-ca998428fcd3.png">

Reply:
I still question the number of files added here. Could this instead be `cypress/` and `!cypress/whatever` the files needed?

cc @bahmutov in there is some context missing.